bin/run_ci_dataset.sh

```diff
@@ -24,9 +24,8 @@
 # Simple script for running an entire dataset through ap_verify
 # Assumes that the requested dataset is already set up in EUPS
 #
-# The ap_verify workspace is a directory with the dataset name; until DM-13887
-# gets resolved, metrics are dumped to the workspace directory with names
-# containing each run's data ID.
+# The ap_verify workspace is a directory with the dataset name; metrics are
+# dumped to the workspace directory with names containing each run's data ID.
```

Comment:
Would it be possible to dump metrics to the output directory so they aren't overwritten if the same dataId is reprocessed later?